src/main/java/duchess/main/DuchessFileHandler.java

```diff
@@ -0,0 +1,139 @@
+package duchess.main;
+
+import duchess.task.Deadline;
+import duchess.task.Event;
+import duchess.task.ToDo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * This class implements a file handler to be used in reading and writing files.
+ *
+ * @author Amos Tan
+ * @version CS2103T AY21/22 Semester 1
+ */
+public class DuchessFileHandler {
+
+    /** The save file of the tasks from Duchess. */
+    public static File savedDuchess;
+    /** The home of the user to search for existing Duchess.*/
+    private static final String DATA_FOLDER = "data";
+    /** The file name of the existing Duchess.*/
+    private static final String FILE_LOCATION = "data/duchess.txt";
+
+    /**
+     * Extracts the DuchessList from the save file.
+     * @param existingDuchessFile The save file.
```

Comment:
Shouldn't you include a empty line between description and parameter? I think this may come out in other areas also